src/geo/gmaps/gmaps-map-view.js

```diff
@@ -70,6 +70,11 @@ var GoogleMapsMapView = MapView.extend({
     this.projector.draw = this._ready;
   },
 
+  clean: function () {
+    google.maps.event.clearInstanceListeners(window);
+    google.maps.event.clearInstanceListeners(document);
```

Comment:
We're overriding `MapView.prototype.clean` here, so we should have invoked `MapView.prototype.clean.call(this);` here. I'm fixing it, but I just wanted to pointed it out since we didn't notice.